curator/theme.json

```diff
@@ -187,6 +187,11 @@
 					"name": "Extra Large",
 					"size": "2rem",
 					"slug": "x-large"
+				},
+				{
+					"name": "Huge",
+					"size": "2.75rem",
```

Comment:
I _think_ we decided limit these font sizes to the sizes that are available in Gutenberg by default. That way switching themes is least likely to "break" a user's decision to map of preset font size to some text content. 

cc @pbking as I always gut check this question with you, and go back and forth every time 😅

Reply:
Yeah I thought I remembered something like this, but I couldn't remember what we decided. Sorry for bringing it up again, now you've mentioned it I'm pretty sure it was to stick with the Gutenberg defaults! I'll fix this up.